metrics/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from topics.models import Article, Topic
+
+
+class Metric(models.Model):
+    article = models.ForeignKey(Article)
+    topic = models.ForeignKey(Topic)
```

Comment:
I assume this is here because an article can have many topics, so we need to know which one has been commented on? I'm wondering whether that's necessary; wouldn't it be more functional for the metric to be seen across all topics?